expression/helper.go

```diff
@@ -56,26 +56,45 @@ func IsValidCurrentTimestampExpr(exprNode ast.ExprNode, fieldType *types.FieldTy
 	return (containsArg && isConsistent) || (!containsArg && !containsFsp)
 }
 
+// ExportedGetTimeCurrentTimestamp is used for generating a timestamp for some special cases: cast null value to timestamp type with not null flag.
+func GetTimeCurrentTimestamp(ctx sessionctx.Context, tp byte, fsp int8) (d types.Datum, err error) {
```

Comment:
`GetTimeCurrentTimestamp` isn't be used. And you use `ExportedGetTimeCurrentTimestamp` instead of this.

Reply:
Please add unit tests for this function.